example/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -363,7 +363,7 @@
 					"$(inherited)",
```

Comment:
readme and changelog must be updated for this.

Reply:
Updated